src/Button/Button.js

```diff
@@ -26,6 +26,8 @@ type Props = {
   circular?: boolean,
   /** Disables the Button */
   disabled?: boolean,
+  /** Element to be used as the root node - e.g. 'a' */
+  element?: string,
```

Comment:
Users, including us, will likely want this to work with ReactRouter Links too.  Have you tested this functionality at all?  I think the `element` Flow type would need to change though.  Unfortunately, I think we use `$FlowFixMe` in our `Link` component at present.

Reply:
I haven't tested this. Just thought about it this morning...

Reply:
this works just fine with an updated flow type